make/testing_flags.mk

```diff
@@ -67,6 +67,10 @@ ifneq ($(tags),)
 DEV_TAGS += ${tags}
 endif
 
+ifneq ($(extratags),)
```

Comment:
I think we only need this for the release tags? Given the lines above will _add_ etcd/postgres based on a name vs the lower level build tag.